electron-builder.json

```diff
@@ -56,7 +55,7 @@
   },
   "fileAssociations": {
     "ext": "vha2",
-    "name": "Video Hub App 2 file",
+    "name": "Video Hub App 3 file",
```

Comment:
Is this strictly true?

It might be a better idea to load `vha2` files, but save them as `vha3` files, as I doubt they're backwards compatible with v2? 🤔 

I completely support keeping the ability to open vha2 files in addition to vha3 files, but if we "corrupt" them from a vha2 point of view, I think that's a bad thing. Thoughts? 💭

Reply:
As of now, `.vha3` simply adds a new property `inputSources` (next to `inputSource`) 🤷 

Unless the hub has more than one input source, it will be safe to open and edit in VHA2 👌 

I was thinking of keeping the file format in case users try VHA3, don't like it, and want to go back to VHA2 (at least until VHA3 is good enough). The way I coded it would make this behavior easiest.